src/org/labkey/targetedms/model/QCPlotFragment.java

```diff
@@ -117,6 +117,10 @@ public JSONObject toJSON(boolean includeLJ, boolean includeMR, boolean includeMe
             {
                 dataJsonObject.put("ValueOutlier", true);
             }
+            if (plotData.isMeanDeviationOutlier(targetedStats.get(plotData.getGuideSetKey())))
```

Comment:
Do we need to return this separate from other value outliers? We're currently using `ValueOutlier` for fixed cutoff or Levey-Jennings, so it seems unexpected to break out mean deviation.